types/ceddl__ceddl-polyfill/index.d.ts

```diff
@@ -0,0 +1,150 @@
+// Type definitions for @ceddl/ceddl-polyfill 0.9
+// Project: https://www.ceddlbyexample.com/
+// Definitions by: Roland Broekema <https://github.com/broekema41>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 4.4
+
+declare module "@ceddl" {}
+
+declare module "@ceddl/ceddl-polyfill" {
```

Comment:
this is scoped module `@ceddl/ceddl-polyfill`, so it should be documented in `tsconfig.json`:
https://github.com/DefinitelyTyped/DefinitelyTyped#what-about-scoped-packages
why do you need `declare module` syntax here? (instead of using exports)

Reply:
Thank you peterblazejewicz. changes made with these hints have made it way better to read.